src/crypto/tls/common.go

```diff
@@ -462,6 +463,12 @@ const (
 	ECDSAWithSHA1 SignatureScheme = 0x0203
 )
 
+type TLSFlag uint16
+
+const (
+	FlagSupportMTLS TLSFlag = 0x50
```

Comment:
Please document what this is. IIRC this is not yet a codepoint that's documented? If so, we should mark it as experimental in the API.
```suggestion
        /// ExperimentalFlagSuppportMTLS ...
	ExperimentalFlagSupportMTLS TLSFlag = 0x50
```

Reply:
Is it intentional that this flag occupies 11 bytes? 0x50 is 80 decimal, and since we start counting at 0, this occupies `ceil(81 / 8) = 11` bytes.

Reply:
Earlier versions of the TLS Flags draft offered guidance on how to select which bit you were using. Following it gave me bit 80. That guidance is now removed so I can select another bit if it's preferred.